api/src/main/java/edu/cornell/mannlib/vitro/webapp/config/ConfigurationPropertiesSmokeTests.java

```diff
@@ -119,14 +123,46 @@ private void checkLanguages(ConfigurationProperties props, StartupStatus ss) {
 		String filterString = props.getProperty(PROPERTY_LANGUAGE_FILTER,
 				"true");
 		boolean languageFilter = Boolean.valueOf(filterString);
+		String i18nDirPath = ctx.getRealPath("/i18n");
 
-		if (selectableLanguages && !buildWithLanguages) {
-			ss.warning(this, String.format("Problem with Language setup - "
-					+ "runtime.properties specifies a "
-					+ "list of selectable languages (%s = %s), but "
-					+ "build.properties did not include any languages with %s",
-					PROPERTY_LANGUAGE_SELECTABLE, selectString,
-					PROPERTY_LANGUAGE_BUILD));
+		if (i18nDirPath == null) {
+			throw new IllegalStateException(
+					"Application does not have an /i18n directory.");
+		}
+
+		List<String> i18nNames = null;
+
+		try {
+		  i18nNames = geti18nNames(i18nDirPath);
+		}
+		catch(java.io.IOException e) {
+		  e.printStackTrace();
+		}
```

Comment:
`e.printStackTrace()` just dumps to the log, and not even to the VIVO log -- goes against the purpose of the smoke test. Handle the exception inside `geti18nNames()`